WcaOnRails/config/locales/locales.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+﻿# frozen_string_literal: true
```

Comment:
Looks like 3 of the files you touched have BOMs that need to be removed. See https://travis-ci.org/thewca/worldcubeassociation.org/builds/196437173#L444-L451:

```
$ bundle exec pre-commit run git
pre-commit: Stopping commit because of errors.
config/pre_commit.rb failed:
WcaOnRails/config/locales/carrierwave.ru.yml:1:1 Found illegal string: byte order marker (BOM)
WcaOnRails/config/locales/locales.rb:1:1 Found illegal string: byte order marker (BOM)
WcaOnRails/config/locales/time_will_tell.ru.yml:1:1 Found illegal string: byte order marker (BOM)
pre-commit: You can bypass this check using `git commit -n`
```